user_docs/en/userGuide.t2t

```diff
@@ -2784,11 +2784,21 @@ You can browse installed incompatible add-ons using the [incompatible add-ons ta
 + Extra Tools +[ExtraTools]
 
 ++ Log Viewer ++[LogViewer]
-The log viewer, found under Tools in the NVDA menu, allows you to view all the logging output that has occurred up until now from when you last started NVDA.

```

Comment:
1. Shouldn't this be "have occurred" instead of "has occurred"?
2. @XLTechie, sorry, I am not sure to have understood your point on this sentence. IMO, removing "up until now " would make the sentence shorter without any loss of meaning. Anyway, if you think otherwise, just let it know.

Reply:
removed "all" from the sentence, it matches better now because it depends on what kind of logging you set in the general settings.

Reply:
> 1. Shouldn't this be "have occurred" instead of "has occurred"?

My mistake; ignore this point.